integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResizeSuite.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.integration.suites
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import tech.beshu.ror.integration.suites.base.support.BaseSingleNodeEsClusterTest
+import tech.beshu.ror.integration.utils.ESVersionSupport
+import tech.beshu.ror.utils.containers.{ElasticsearchNodeDataInitializer, EsContainerCreator}
+import tech.beshu.ror.utils.elasticsearch.IndexManager.ReindexSource
+import tech.beshu.ror.utils.elasticsearch.{DocumentManager, IndexManager}
+import tech.beshu.ror.utils.httpclient.RestClient
+
+trait ResizeSuite
+  extends AnyWordSpec
+    with BaseSingleNodeEsClusterTest
+    with ESVersionSupport
+    with Matchers {
+  this: EsContainerCreator =>
+
+  override implicit val rorConfigFileName = "/resize/readonlyrest.yml"
+
+  private lazy val user1IndexManager = new IndexManager(basicAuthClient("dev1", "test"))
+
+  override def nodeDataInitializer = Some(ResizeSuite.nodeDataInitializer())
+
+  "A resize request" should {
+    "be able to proceed" when {
+      "user has permission to source index and dest index"  in {
+        val result = user1IndexManager.resize("test1_index", "test1_index_resized")
+
+        result.isSuccess should be(true)
```

Comment:
IMO it's better to check a response code here and below

Reply:
I also changed it in `RemoteReindexSuite` and `ReindexSuite` because it was handled the same way